.github/workflows/main_eagleapi.yml

```diff
@@ -0,0 +1,57 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app to Azure Web App - EagleApi
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:

```

Comment:
It doesn't look like you're doing any kind of `dotnet test` in your pipeline to run your tests on deployment?